app/domain/logs.rb

```diff
@@ -591,106 +591,106 @@ module AuthnJwt
       )
 
       ClaimsDenyListValue = ::Util::TrackableLogMessageClass.new(
-        msg: "Claims denylist value is '{0-deny-claims-list}'",
+        msg:  "Claims denylist value is '{0-deny-claims-list}'",
```

Comment:
Please provide filled in example of message

Reply:
for example `iss, exp, nbf, ist`

Reply:
I'm not sure about this sentence.
maybe it should be:
The claims {0-deny-claims-list} was added to denylist

The claims `iss, exp, nbf, ist` was added to denylist

@shulifink  WDYT?

Reply:
```suggestion
        msg:  "The following claims have been added to the denylist: '{0-deny-claims-list}'",
```

Reply:
@hilagross @shulifink 
```suggestion
        msg:  "The following claims are in denylist: '{0-deny-claims-list}'",
```
WDYT?